public/externals.json

```diff
@@ -1,6 +1,10 @@
 {
   "commons": {
     "dev": [
+      {
+        "path": "https://unpkg.com/bluebird@3.5.1/js/browser/bluebird.js",
```

Comment:
Is the `browser` version the one we want for this case? I guess it overrides window.Promise, right? In this case, LGTM

Reply:
yes, it overrides window.Promise :)